health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/MapViewPopup.js

```
@@ -0,0 +1,333 @@
+</svg>
+    `,
+    iconSize: [40, 40], // Adjust based on your SVG size
+    iconAnchor: [20, 40], // Positioning so it aligns properly
+    popupAnchor: [0, -40],
+  });
+
+  // Function to initialize map
+  const init = (id, defaultBaseMap) => {
+    if (mapRef.current) return;
+
+    let mapConfig = {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'mapConfig' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)